taxcalc/functions.py

```diff
@@ -184,11 +185,18 @@ def CapGains(p23250, p22250, _sep, ALD_Investment_ec, ALD_StudentLoan_HC,
     c23650 = p23250 + p22250
     # limitation on capital losses
     c01000 = max((-3000. / _sep), c23650)
-    # compute ymod* variables
+    # compute ymod1 variable that is included in AGI
     ymod1 = (e00200 + e00700 + e00800 + e00900 + e01400 + e01700 +
              (1 - ALD_Investment_ec) * (e00300 + e00600 +
                                         c01000 + e01100 + e01200) +
              e02000 + e02100 + e02300)
+    if CG_nodiff != 0.:
```

Comment:
Yes.  Look at the sponsor's description of the reform in the first item in this discussion.

Both current law and this reform proposal give capital gains and qualified dividends preferential treatment, but they do that in different ways.  Current law taxes **long-term** capital gain and qualified dividends at lower rates than other income, while this proposal taxes them at the same rates as other income but gives allows an exclusion of **all** capital gains and qualified dividends.  Neither current law nor this reform proposal contemplates allowing exclusions and lower tax rates.